[2025-04-23T20:02:41.899+0000] {processor.py:161} INFO - Started process (PID=169) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:02:41.900+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:02:41.900+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:02:41.900+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:02:41.914+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:02:41.912+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
ModuleNotFoundError: No module named 'ingestion_DAG'
[2025-04-23T20:02:41.914+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:02:41.930+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.034 seconds
[2025-04-23T20:03:12.250+0000] {processor.py:161} INFO - Started process (PID=174) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:03:12.251+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:03:12.252+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:03:12.252+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:03:12.260+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:03:12.260+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
ModuleNotFoundError: No module named 'ingestion_DAG'
[2025-04-23T20:03:12.261+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:03:12.290+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.045 seconds
[2025-04-23T20:03:42.551+0000] {processor.py:161} INFO - Started process (PID=179) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:03:42.554+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:03:42.556+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:03:42.555+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:03:42.592+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:03:42.578+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
ModuleNotFoundError: No module named 'ingestion_DAG'
[2025-04-23T20:03:42.593+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:03:42.729+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.184 seconds
[2025-04-23T20:04:12.954+0000] {processor.py:161} INFO - Started process (PID=184) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:04:12.957+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:04:12.959+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:04:12.958+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:04:12.970+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:04:12.969+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
ModuleNotFoundError: No module named 'ingestion_DAG'
[2025-04-23T20:04:12.970+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:04:13.002+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.053 seconds
[2025-04-23T20:04:43.294+0000] {processor.py:161} INFO - Started process (PID=189) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:04:43.295+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:04:43.296+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:04:43.296+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:04:43.305+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:04:43.304+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
ModuleNotFoundError: No module named 'ingestion_DAG'
[2025-04-23T20:04:43.305+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:04:43.333+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.044 seconds
[2025-04-23T20:05:13.656+0000] {processor.py:161} INFO - Started process (PID=194) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:05:13.658+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:05:13.660+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:05:13.659+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:05:13.669+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:05:13.668+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
ModuleNotFoundError: No module named 'ingestion_DAG'
[2025-04-23T20:05:13.669+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:05:13.693+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.042 seconds
[2025-04-23T20:05:43.998+0000] {processor.py:161} INFO - Started process (PID=199) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:05:44.001+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:05:44.002+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:05:44.002+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:05:44.012+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:05:44.011+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
ModuleNotFoundError: No module named 'ingestion_DAG'
[2025-04-23T20:05:44.012+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:05:44.034+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.042 seconds
[2025-04-23T20:06:14.419+0000] {processor.py:161} INFO - Started process (PID=204) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:06:14.420+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:06:14.422+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:06:14.421+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:06:14.430+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:06:14.429+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
ModuleNotFoundError: No module named 'ingestion_DAG'
[2025-04-23T20:06:14.430+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:06:14.459+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.046 seconds
[2025-04-23T20:06:44.814+0000] {processor.py:161} INFO - Started process (PID=209) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:06:44.816+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:06:44.818+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:06:44.817+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:06:44.826+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:06:44.825+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
ModuleNotFoundError: No module named 'ingestion_DAG'
[2025-04-23T20:06:44.827+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:06:44.859+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.052 seconds
[2025-04-23T20:07:15.240+0000] {processor.py:161} INFO - Started process (PID=214) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:07:15.243+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:07:15.244+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:07:15.244+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:07:15.254+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:07:15.253+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
ModuleNotFoundError: No module named 'ingestion_DAG'
[2025-04-23T20:07:15.255+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:07:15.282+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.047 seconds
[2025-04-23T20:07:45.631+0000] {processor.py:161} INFO - Started process (PID=219) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:07:45.633+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:07:45.635+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:07:45.634+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:07:45.644+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:07:45.643+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
ModuleNotFoundError: No module named 'ingestion_DAG'
[2025-04-23T20:07:45.645+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:07:45.670+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.045 seconds
[2025-04-23T20:08:16.003+0000] {processor.py:161} INFO - Started process (PID=224) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:08:16.005+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:08:16.008+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:08:16.006+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:08:16.019+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:08:16.018+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
ModuleNotFoundError: No module named 'ingestion_DAG'
[2025-04-23T20:08:16.019+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:08:16.047+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.051 seconds
[2025-04-23T20:08:46.339+0000] {processor.py:161} INFO - Started process (PID=229) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:08:46.341+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:08:46.342+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:08:46.342+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:08:46.353+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:08:46.351+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
ModuleNotFoundError: No module named 'ingestion_DAG'
[2025-04-23T20:08:46.353+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:08:46.372+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.035 seconds
[2025-04-23T20:09:16.639+0000] {processor.py:161} INFO - Started process (PID=234) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:09:16.641+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:09:16.642+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:09:16.641+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:09:16.650+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:09:16.649+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
ModuleNotFoundError: No module named 'ingestion_DAG'
[2025-04-23T20:09:16.651+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:09:16.678+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.045 seconds
[2025-04-23T20:09:46.947+0000] {processor.py:161} INFO - Started process (PID=239) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:09:46.949+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:09:46.950+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:09:46.949+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:09:46.958+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:09:46.957+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
ModuleNotFoundError: No module named 'ingestion_DAG'
[2025-04-23T20:09:46.958+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:09:46.979+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.037 seconds
[2025-04-23T20:12:24.334+0000] {processor.py:161} INFO - Started process (PID=161) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:12:24.335+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:12:24.336+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:12:24.336+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:12:24.347+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:12:24.345+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
ModuleNotFoundError: No module named 'ingestion_DAG'
[2025-04-23T20:12:24.347+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:12:24.365+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.034 seconds
[2025-04-23T20:12:54.598+0000] {processor.py:161} INFO - Started process (PID=166) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:12:54.599+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:12:54.600+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:12:54.600+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:12:54.607+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:12:54.606+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
ModuleNotFoundError: No module named 'ingestion_DAG'
[2025-04-23T20:12:54.608+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:12:54.631+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.037 seconds
[2025-04-23T20:13:24.900+0000] {processor.py:161} INFO - Started process (PID=171) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:13:24.901+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:13:24.902+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:13:24.902+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:13:24.909+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:13:24.908+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
ModuleNotFoundError: No module named 'ingestion_DAG'
[2025-04-23T20:13:24.909+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:13:24.932+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.036 seconds
[2025-04-23T20:13:55.176+0000] {processor.py:161} INFO - Started process (PID=176) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:13:55.177+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:13:55.177+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:13:55.177+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:13:55.185+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:13:55.184+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
ModuleNotFoundError: No module named 'ingestion_DAG'
[2025-04-23T20:13:55.185+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:13:55.222+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.054 seconds
[2025-04-23T20:14:25.483+0000] {processor.py:161} INFO - Started process (PID=181) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:14:25.485+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:14:25.486+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:14:25.485+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:14:25.495+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:14:25.494+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
ModuleNotFoundError: No module named 'ingestion_DAG'
[2025-04-23T20:14:25.496+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:14:25.527+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.054 seconds
[2025-04-23T20:14:55.801+0000] {processor.py:161} INFO - Started process (PID=186) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:14:55.802+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:14:55.803+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:14:55.803+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:14:55.814+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:14:55.813+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
ModuleNotFoundError: No module named 'ingestion_DAG'
[2025-04-23T20:14:55.815+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:14:55.842+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.046 seconds
[2025-04-23T20:15:26.105+0000] {processor.py:161} INFO - Started process (PID=191) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:15:26.106+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:15:26.107+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:15:26.106+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:15:26.119+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:15:26.118+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
ModuleNotFoundError: No module named 'ingestion_DAG'
[2025-04-23T20:15:26.119+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:15:26.145+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.046 seconds
[2025-04-23T20:16:00.713+0000] {processor.py:161} INFO - Started process (PID=192) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:16:00.714+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:16:00.714+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:16:00.714+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:16:00.718+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:16:00.717+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:16:00.718+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:16:00.729+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.019 seconds
[2025-04-23T20:16:31.144+0000] {processor.py:161} INFO - Started process (PID=224) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:16:31.145+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:16:31.145+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:16:31.145+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:16:31.149+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:16:31.148+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:16:31.149+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:16:31.160+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.017 seconds
[2025-04-23T20:17:01.568+0000] {processor.py:161} INFO - Started process (PID=252) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:17:01.569+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:17:01.570+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:17:01.570+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:17:01.575+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:17:01.573+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:17:01.577+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:17:01.591+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.024 seconds
[2025-04-23T20:17:32.080+0000] {processor.py:161} INFO - Started process (PID=282) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:17:32.082+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:17:32.082+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:17:32.082+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:17:32.087+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:17:32.087+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:17:32.088+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:17:32.100+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.022 seconds
[2025-04-23T20:18:02.733+0000] {processor.py:161} INFO - Started process (PID=314) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:18:02.734+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:18:02.734+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:18:02.734+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:18:02.739+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:18:02.738+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:18:02.739+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:18:02.751+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.020 seconds
[2025-04-23T20:18:27.035+0000] {processor.py:161} INFO - Started process (PID=319) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:18:27.039+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:18:27.041+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:18:27.040+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:18:27.068+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:18:27.067+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:18:27.076+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:18:27.114+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.084 seconds
[2025-04-23T20:18:28.052+0000] {processor.py:161} INFO - Started process (PID=324) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:18:28.054+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:18:28.054+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:18:28.054+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:18:28.061+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:18:28.060+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:18:28.061+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:18:28.079+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.030 seconds
[2025-04-23T20:18:58.609+0000] {processor.py:161} INFO - Started process (PID=354) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:18:58.612+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:18:58.613+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:18:58.613+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:18:58.627+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:18:58.625+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:18:58.628+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:18:58.650+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.047 seconds
[2025-04-23T20:19:29.110+0000] {processor.py:161} INFO - Started process (PID=384) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:19:29.116+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:19:29.119+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:19:29.117+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:19:29.142+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:19:29.138+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:19:29.142+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:19:29.171+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.069 seconds
[2025-04-23T20:22:51.590+0000] {processor.py:161} INFO - Started process (PID=161) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:22:51.597+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:22:51.598+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:22:51.598+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:22:51.627+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:22:51.622+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:22:51.628+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:22:51.661+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.074 seconds
[2025-04-23T20:23:22.386+0000] {processor.py:161} INFO - Started process (PID=191) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:23:22.388+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:23:22.391+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:23:22.390+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:23:22.407+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:23:22.406+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:23:22.407+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:23:22.436+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.073 seconds
[2025-04-23T20:23:52.981+0000] {processor.py:161} INFO - Started process (PID=221) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:23:52.983+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:23:52.989+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:23:52.985+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:23:53.008+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:23:53.007+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:23:53.009+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:23:53.051+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.078 seconds
[2025-04-23T20:24:23.772+0000] {processor.py:161} INFO - Started process (PID=251) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:24:23.774+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:24:23.777+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:24:23.777+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:24:23.792+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:24:23.790+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:24:23.794+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:24:23.835+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.074 seconds
[2025-04-23T20:24:54.330+0000] {processor.py:161} INFO - Started process (PID=281) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:24:54.335+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:24:54.336+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:24:54.336+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:24:54.350+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:24:54.347+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:24:54.351+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:24:54.379+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.055 seconds
[2025-04-23T20:25:24.886+0000] {processor.py:161} INFO - Started process (PID=311) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:25:24.889+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:25:24.895+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:25:24.894+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:25:24.922+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:25:24.918+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:25:24.923+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:25:24.956+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.086 seconds
[2025-04-23T20:26:08.344+0000] {processor.py:161} INFO - Started process (PID=161) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:26:08.347+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:26:08.349+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:26:08.348+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:26:08.365+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:26:08.364+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:26:08.366+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:26:08.402+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.074 seconds
[2025-04-23T20:26:38.805+0000] {processor.py:161} INFO - Started process (PID=191) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:26:38.807+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:26:38.809+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:26:38.809+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:26:38.840+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:26:38.838+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:26:38.841+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:26:38.865+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.065 seconds
[2025-04-23T20:27:09.410+0000] {processor.py:161} INFO - Started process (PID=221) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:27:09.414+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:27:09.417+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:27:09.417+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:27:09.428+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:27:09.427+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:27:09.431+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:27:09.466+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.062 seconds
[2025-04-23T20:27:39.936+0000] {processor.py:161} INFO - Started process (PID=251) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:27:39.947+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:27:39.953+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:27:39.951+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:27:39.963+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:27:39.961+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:27:39.966+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:27:39.990+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.068 seconds
[2025-04-23T20:28:10.396+0000] {processor.py:161} INFO - Started process (PID=281) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:28:10.398+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:28:10.401+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:28:10.401+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:28:10.416+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:28:10.413+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:28:10.416+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:28:10.442+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.059 seconds
[2025-04-23T20:28:40.928+0000] {processor.py:161} INFO - Started process (PID=311) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:28:40.932+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:28:40.934+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:28:40.934+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:28:40.950+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:28:40.948+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:28:40.951+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:28:40.986+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.069 seconds
[2025-04-23T20:29:11.521+0000] {processor.py:161} INFO - Started process (PID=341) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:29:11.526+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:29:11.529+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:29:11.528+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:29:11.547+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:29:11.546+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:29:11.549+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:29:11.578+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.066 seconds
[2025-04-23T20:29:42.139+0000] {processor.py:161} INFO - Started process (PID=371) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:29:42.144+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:29:42.147+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:29:42.146+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:29:42.169+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:29:42.166+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:29:42.170+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:29:42.213+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.090 seconds
[2025-04-23T20:30:12.829+0000] {processor.py:161} INFO - Started process (PID=401) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:30:12.833+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:30:12.837+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:30:12.835+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:30:12.867+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:30:12.862+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:30:12.868+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:30:12.938+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.122 seconds
[2025-04-23T20:30:43.538+0000] {processor.py:161} INFO - Started process (PID=431) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:30:43.541+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:30:43.543+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:30:43.542+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:30:43.563+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:30:43.561+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:30:43.566+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:30:43.595+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.065 seconds
[2025-04-23T20:31:14.172+0000] {processor.py:161} INFO - Started process (PID=461) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:31:14.175+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:31:14.177+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:31:14.176+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:31:14.198+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:31:14.194+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:31:14.199+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:31:14.224+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.068 seconds
[2025-04-23T20:31:44.635+0000] {processor.py:161} INFO - Started process (PID=491) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:31:44.637+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:31:44.639+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:31:44.639+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:31:44.651+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:31:44.648+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:31:44.652+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:31:44.672+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.042 seconds
[2025-04-23T20:32:15.327+0000] {processor.py:161} INFO - Started process (PID=521) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:32:15.332+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:32:15.335+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:32:15.334+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:32:15.364+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:32:15.357+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:32:15.367+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:32:15.468+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.161 seconds
[2025-04-23T20:32:45.963+0000] {processor.py:161} INFO - Started process (PID=551) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:32:45.969+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:32:45.970+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:32:45.970+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:32:45.991+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:32:45.990+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:32:45.994+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:32:46.038+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.090 seconds
[2025-04-23T20:33:16.499+0000] {processor.py:161} INFO - Started process (PID=581) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:33:16.501+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:33:16.505+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:33:16.504+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:33:16.521+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:33:16.520+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:33:16.523+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:33:16.568+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.081 seconds
[2025-04-23T20:33:47.046+0000] {processor.py:161} INFO - Started process (PID=611) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:33:47.050+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:33:47.052+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:33:47.051+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:33:47.066+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:33:47.061+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:33:47.067+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:33:47.104+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.070 seconds
[2025-04-23T20:34:17.609+0000] {processor.py:161} INFO - Started process (PID=641) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:34:17.611+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:34:17.613+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:34:17.612+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:34:17.627+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:34:17.626+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:34:17.628+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:34:17.666+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.069 seconds
[2025-04-23T20:34:48.178+0000] {processor.py:161} INFO - Started process (PID=671) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:34:48.181+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:34:48.182+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:34:48.182+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:34:48.198+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:34:48.195+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:34:48.199+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:34:48.224+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.058 seconds
[2025-04-23T20:35:18.479+0000] {processor.py:161} INFO - Started process (PID=701) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:35:18.481+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:35:18.482+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:35:18.482+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:35:18.493+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:35:18.492+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:35:18.493+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:35:18.519+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.047 seconds
[2025-04-23T20:50:26.631+0000] {processor.py:161} INFO - Started process (PID=731) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:50:26.633+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:50:26.634+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:50:26.634+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:50:26.643+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:50:26.641+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:50:26.644+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:50:26.668+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.043 seconds
[2025-04-23T20:50:57.213+0000] {processor.py:161} INFO - Started process (PID=763) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:50:57.216+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:50:57.218+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:50:57.217+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:50:57.235+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:50:57.232+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:50:57.236+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:50:57.272+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.070 seconds
[2025-04-23T20:51:27.746+0000] {processor.py:161} INFO - Started process (PID=793) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:51:27.749+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:51:27.751+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:51:27.751+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:51:27.770+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:51:27.765+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:51:27.770+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:51:27.807+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.071 seconds
[2025-04-23T20:51:58.287+0000] {processor.py:161} INFO - Started process (PID=823) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:51:58.290+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:51:58.292+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:51:58.292+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:51:58.301+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:51:58.299+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:51:58.301+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:51:58.325+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.043 seconds
[2025-04-23T20:52:28.860+0000] {processor.py:161} INFO - Started process (PID=853) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:52:28.863+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T20:52:28.865+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:52:28.864+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:52:28.882+0000] {logging_mixin.py:188} INFO - [2025-04-23T20:52:28.881+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T20:52:28.882+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T20:52:28.917+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.067 seconds
[2025-04-23T21:07:56.086+0000] {processor.py:161} INFO - Started process (PID=885) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T21:07:56.089+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T21:07:56.092+0000] {logging_mixin.py:188} INFO - [2025-04-23T21:07:56.089+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T21:07:56.107+0000] {logging_mixin.py:188} INFO - [2025-04-23T21:07:56.105+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T21:07:56.107+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T21:07:56.173+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.111 seconds
[2025-04-23T21:08:26.840+0000] {processor.py:161} INFO - Started process (PID=915) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T21:08:26.843+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T21:08:26.845+0000] {logging_mixin.py:188} INFO - [2025-04-23T21:08:26.844+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T21:08:26.860+0000] {logging_mixin.py:188} INFO - [2025-04-23T21:08:26.858+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T21:08:26.861+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T21:08:26.891+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.078 seconds
[2025-04-23T21:23:53.282+0000] {processor.py:161} INFO - Started process (PID=947) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T21:23:53.285+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T21:23:53.287+0000] {logging_mixin.py:188} INFO - [2025-04-23T21:23:53.286+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T21:23:53.306+0000] {logging_mixin.py:188} INFO - [2025-04-23T21:23:53.302+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T21:23:53.306+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T21:23:53.344+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.069 seconds
[2025-04-23T21:39:21.948+0000] {processor.py:161} INFO - Started process (PID=977) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T21:39:21.956+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T21:39:21.975+0000] {logging_mixin.py:188} INFO - [2025-04-23T21:39:21.962+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T21:39:22.022+0000] {logging_mixin.py:188} INFO - [2025-04-23T21:39:22.018+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T21:39:22.023+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T21:39:22.113+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.184 seconds
[2025-04-23T21:39:52.789+0000] {processor.py:161} INFO - Started process (PID=1007) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T21:39:52.793+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T21:39:52.794+0000] {logging_mixin.py:188} INFO - [2025-04-23T21:39:52.794+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T21:39:52.809+0000] {logging_mixin.py:188} INFO - [2025-04-23T21:39:52.808+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T21:39:52.809+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T21:39:52.845+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.073 seconds
[2025-04-23T21:55:19.369+0000] {processor.py:161} INFO - Started process (PID=1037) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T21:55:19.375+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T21:55:19.377+0000] {logging_mixin.py:188} INFO - [2025-04-23T21:55:19.376+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T21:55:19.391+0000] {logging_mixin.py:188} INFO - [2025-04-23T21:55:19.387+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T21:55:19.391+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T21:55:19.432+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.075 seconds
[2025-04-23T21:55:49.929+0000] {processor.py:161} INFO - Started process (PID=1067) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T21:55:49.931+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T21:55:49.933+0000] {logging_mixin.py:188} INFO - [2025-04-23T21:55:49.932+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T21:55:49.955+0000] {logging_mixin.py:188} INFO - [2025-04-23T21:55:49.954+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T21:55:49.957+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T21:55:49.984+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.061 seconds
[2025-04-23T22:35:25.528+0000] {processor.py:161} INFO - Started process (PID=1099) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T22:35:25.530+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T22:35:25.538+0000] {logging_mixin.py:188} INFO - [2025-04-23T22:35:25.535+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T22:35:25.560+0000] {logging_mixin.py:188} INFO - [2025-04-23T22:35:25.556+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T22:35:25.561+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T22:35:25.656+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.134 seconds
[2025-04-23T23:03:24.041+0000] {processor.py:161} INFO - Started process (PID=1129) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T23:03:24.045+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T23:03:24.046+0000] {logging_mixin.py:188} INFO - [2025-04-23T23:03:24.046+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T23:03:24.060+0000] {logging_mixin.py:188} INFO - [2025-04-23T23:03:24.056+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T23:03:24.060+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T23:03:24.091+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.057 seconds
[2025-04-23T23:36:35.886+0000] {processor.py:161} INFO - Started process (PID=1159) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T23:36:35.887+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T23:36:35.890+0000] {logging_mixin.py:188} INFO - [2025-04-23T23:36:35.888+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T23:36:35.907+0000] {logging_mixin.py:188} INFO - [2025-04-23T23:36:35.905+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T23:36:35.907+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T23:36:35.983+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.104 seconds
[2025-04-23T23:39:27.983+0000] {processor.py:161} INFO - Started process (PID=1189) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T23:39:27.985+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T23:39:27.988+0000] {logging_mixin.py:188} INFO - [2025-04-23T23:39:27.987+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T23:39:28.001+0000] {logging_mixin.py:188} INFO - [2025-04-23T23:39:27.998+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T23:39:28.001+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T23:39:28.029+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.055 seconds
[2025-04-23T23:43:44.326+0000] {processor.py:161} INFO - Started process (PID=1221) to work on /opt/airflow/dags/Master_DAG.py
[2025-04-23T23:43:44.359+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Master_DAG.py for tasks to queue
[2025-04-23T23:43:44.367+0000] {logging_mixin.py:188} INFO - [2025-04-23T23:43:44.366+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/Master_DAG.py
[2025-04-23T23:43:44.549+0000] {logging_mixin.py:188} INFO - [2025-04-23T23:43:44.539+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/Master_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Master_DAG.py", line 12, in <module>
    from ingestion_DAG import process_json_file
  File "/opt/airflow/dags/ingestion_DAG.py", line 7, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-04-23T23:43:44.550+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/Master_DAG.py
[2025-04-23T23:43:44.752+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Master_DAG.py took 0.439 seconds
